src/depackers/vorbis.c

```diff
@@ -3320,10 +3304,6 @@ static int vorbis_decode_packet_rest(vorb *f, int *len, Mode *m, int left_start,
             }
 
 #ifdef STB_VORBIS_NO_DEFER_FLOOR
-	    /* Sanity check */
-	    if (n >= f->blocksize_1)
-		return FALSE;
-
```

Comment:
An easy one for a change:

1) `n` = `f->blocksize[m->blockflag]` -> `n` is either `blocksize_0` or `blocksize_1`.
2) `blocksize_0` is always less than `blocksize_1`. (line 3696)
3) This case will never occur or, worst case, will filter valid floor lengths of `blocksize_1` due to off-by-one.